gammapy/scripts/image_pipe.py

```diff
@@ -101,6 +102,42 @@ def bkg_map(self, bkg_norm=True):
 
         self.maps["bkg"] = bkg_map
 
+    def make_1d_expected_counts(self, spectral_index=2.3, for_integral_flux=False):
+        """Compute the 1D exposure table for one observation for an offset table
+
+        Parameters
+        ----------
+        spectral_index : float
+            Assumed power-law spectral index
+        for_integral_flux : bool
+            True if you want that the total excess / exposure gives the integrated flux
+
+        Returns
+        -------
+        table : `astropy.table.Table`
+            Two columns: offset in the FOV "theta" and expected counts "npred"
+
+        """
+        energy = EnergyBounds.equal_log_spacing(self.energy_band[0].value, self.energy_band[1].value, 100,
+                                                self.energy_band.unit)
+        energy_band = energy.bands
+        energy_bin = energy.log_centers
+        eref = EnergyBounds(self.energy_band).log_centers
+        spectrum = (energy_bin / eref) ** (-spectral_index)
+        offset_tab = Angle(np.linspace(self.offset_band[0].value, self.offset_band[1].value, 10), self.offset_band.unit)
+        arf = self.aeff.evaluate(offset=offset_tab, energy=energy_bin).T
+        npred_tab = np.sum(arf * spectrum * energy_band, axis=1)
+        npred_tab *= self.livetime
+        if for_integral_flux:
+            norm = np.sum(spectrum * energy_band)
+            npred_tab /= norm
+        table = Table()
```

Comment:
Can you change to this?
```
from astropy.table import QTable # put this at the top of the file
table = QTable()
table['theta'] = offset_tab
table['npred'] = npred_tab
```
`QTable` is a table that can store quantities as columns.
And there's no need to create Column objects first, they will be created on `table['colname'] = ...`.